### PR TITLE
changing lines

### PR DESCRIPTION
**Pull request by Mo-chan**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2166)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: arabic/passion/passion.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+إن اردت أن تعيش حياة مليئه بالسعادة٫ عليك اولا ان تجد ما تحب عمله في الحياه
```
Comment (casey-collab) [inline arabic/passion/passion.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: arabic/salamoAlikom.md [hunk 2]
```diff
@@ -9,3 +9,5 @@
 [اريد ان اتعلم المزيد](learn/learn.md)
 
 [اريد ان احلم](dream/dream.md)
+
+(passion/passion.md) [اريد ان اجد شغفي في الحياه]
```
Comment (cbuckey-uda) [inline arabic/salamoAlikom.md]: It looks like you intended this to be a link, but it needs to go in the other order (like the links above). So the text in square brackets should come first, and the link in parentheses should follow.

Comment (cbuckey-uda) [inline arabic/salamoAlikom.md]: By the way, I'm not sure if this happens for you, but my text editor displays Arabic text backwards compared to how it is displayed on GitHub, so perhaps that is the source of the problem? Regardless, if you format this link like the links above it, it should work.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**